src/transformers/modeling_utils.py

```diff
@@ -2796,10 +2798,13 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 offload_state_dict=offload_state_dict,
                 dtype=torch_dtype,
                 load_in_8bit=load_in_8bit,
+                load_in_4bit=load_in_4bit,
                 keep_in_fp32_modules=keep_in_fp32_modules,
             )
 
+        model.is_loaded_in_4bit = load_in_4bit
         model.is_loaded_in_8bit = load_in_8bit
+        model.is_loaded_in_kbit = load_in_8bit or load_in_4bit
```

Comment:
Should we create an intermediate variable  above `is_loaded_in_kbit = load_in_8bit or load_in_4bit` and use all the way down?